test/e2e/multihoming.go

```diff
@@ -27,6 +27,14 @@ import (
 
 const PolicyForAnnotation = "k8s.v1.cni.cncf.io/policy-for"
 
+type Permission int
+
+const (
+	Specific Permission = iota
```

Comment:
why `specific` ? what's its purpose ? it's really not immediately clear.

Reply:
it is not "allow all" / "deny all", i can change it to IPBlock but then also need to see if there are more usages and add one for them as well

(the resolution we care about the permission flag itself is just to know if it is allow / deny all / other)

Reply:
so `specific` means something that is not allow or deny all ? 

That name is really not intuitive. 

`ipBlock` will not work long term, since we want to add support for namespace / pod selectors. 
 
maybe splitting the table into successful / blocked tables will allow you to remove the need for it. 

Wondering if we can do that without duplicating a lof of the test code inside the table function.

Reply:
lets try to split